internal/oci/container.go

```diff
@@ -344,6 +344,23 @@ func (c *Container) exitFilePath() string {
 	return filepath.Join(c.dir, "exit")
 }
 
+// IsRunning is a function that checks if a specific pid is running
```

Comment:
> if a specific pid is running

This is not a good choice of words. If by "pid" you mean a process, it can be in present but in a few other states than "running" (e.g. waiting for I/O to finish, traced, etc.).

I think it would be more correct to say "if a container's init pid exists" (or "present").

Reply:
adapted as suggested!